packages/fluentui/react-northstar/src/themes/teams/colors.ts

```diff
@@ -217,8 +217,8 @@ export const colorScheme: ColorSchemeMapping<ColorScheme, TeamsColorNames> = {
     foreground2: colors.grey[450],
     foreground3: colors.white,
     foreground4: colors.white,
-    foreground5: colors.grey[500],
-    foreground6: colors.grey[600],
+    foreground5: colors.grey[100],
+    foreground6: colors.grey[200],
```

Comment:
Please could you add a note in the Changelog that we are changing the foreground defaults? :)